website/blog/2024-08-18-introduce-tabby-inline-edit/index.mdx

```diff
@@ -0,0 +1,94 @@
+---
+authors: [wei]
+tags: [editor integration, product]
+image: ./tabby-edit.png
+
+---
+
+
+# Introduce the Inline Edit in Tabby
+
+![Tabby Edit](./tabby-edit.png)
+
+## What is Tabby Edit?  
+  
+Tabby Edit is a feature that allows you to edit your code directly in your favorite editor.  
+  
+Tabby Edit will not change your code directly,  
```

Comment:
The key of this feature is to offer a new user interaction -  to enable users to instruct LLM for code changes on a specific piece of code snippet (unlike code completion or chat interface). Would be good to highlight this aspect a bit in the intro section.